doc/source/whatsnew/v1.1.0.rst

```diff
@@ -819,7 +866,6 @@ Groupby/resample/rolling
 - Bug in :meth:`Series.groupby` would raise ``ValueError`` when grouping by :class:`PeriodIndex` level (:issue:`34010`)
 - Bug in :meth:`GroupBy.agg`, :meth:`GroupBy.transform`, and :meth:`GroupBy.resample` where subclasses are not preserved (:issue:`28330`)
 - Bug in :meth:`GroupBy.rolling.apply` ignores args and kwargs parameters (:issue:`33433`)
-- Bug in :meth:`DataFrameGroupby.std` and :meth:`DataFrameGroupby.sem` would modify grouped-by columns when ``as_index=False`` (:issue:`10355`)
```

Comment:
was this removed on purpose?

Reply:
or if this is incorporated in your patch (looks likey), then just add the issue number in your whatsnew (next to the other one)

Reply:
Right - this is incorporated. I added the issue number in this line:

https://github.com/pandas-dev/pandas/pull/34012/files#diff-e763086372257ed7ca944d8bc5042a04R593

(sorry - I can't seem to permalink)